set.mm

```diff
@@ -199622,22 +199694,30 @@ C_ dom ( S sSet <. I , E >. ) ) $=
   starvid $p |- *r = Slot ( *r ` ndx ) $=
     ( cstv c4 df-starv 4nn ndxid ) ABCDE $.
 
+  $( The slot for the involution function is not the slot for the base set in
+     an extensible structure.  Formerly part of proof for ~ ressstarv .
+     (Contributed by AV, 18-Oct-2024.) $)
+  starvndxnbasendx $p |- ( *r ` ndx ) =/= ( Base ` ndx ) $=
+    ( cnx cstv cfv cbs wne c4 c1 1re 1lt4 gtneii starvndx basendx neeq12i mpbir
+    ) ABCZADCZEFGEGFHIJOFPGKLMN $.
+
   ${
     ressmulr.1 $e |- S = ( R |`s A ) $.
     ${
       ressmulr.2 $e |- .x. = ( .r ` R ) $.
       $( ` .r ` is unaffected by restriction.  (Contributed by Stefan O'Rear,
          27-Nov-2014.) $)
       ressmulr $p |- ( A e. V -> .x. = ( .r ` S ) ) $=
-        ( cmulr c3 df-mulr 3nn 1lt3 resslem ) ADCHIEBFGJKLM $.
+        ( cmulr mulrid cnx cbs cfv basendxnmulrndx necomi resseqnbas ) ADCHEBFG
```

Comment:
Perhaps resseqnbas, starvndxnbasendx et al should be commuted to avoid necomi and be consistent with preexisting theorems basendxnmulrndx and slotsbhcdif?

Reply:
> Perhaps resseqnbas, starvndxnbasendx et al should be commuted to avoid necomi and be consistent with preexisting theorems basendxnmulrndx and slotsbhcdif? Some theorems are longer because of the necomi step

I have thought about this, too. But I am not sure about the most appropriate order. For new theorems, I decided to have the index with the higher (numeric) value first. If this turns out to be the right way, I would revise ~basendxnmulrndx etc., otherwise ~starvndxnbasendx. But I want to check this at the end of my work, so please be patient.